pandas/tests/io/test_excel.py

```diff
@@ -2490,3 +2499,71 @@ def custom_converter(css):
             n_cells += 1
 
     assert n_cells == (10 + 1) * (3 + 1)
+
```

Comment:
the code org and the test org ideally will mirror each other. but they can be moved independently. for now I want smaller / more well defined / does 1 thing test files. I think this is a nice way to organize things into chunks. Then can prob tackle the code reorg.

The *formatting* code lives in pandas/io/formats/*, and style fits right in here. 
Style shouldn't be directly connected (nor depended on in any way) in the core code. It is simply output formatting.